### ISSUE TITLE
[CLN] Collection of code clean-up and improvement tasks

### ISSUE THREAD

**Issue by theOehrly** [theOehrly/Fast-F1#346]
This issue is intended for keeping track of some code clean-up and improvement tasks that are not immediately required but should be done at some point.

In no particular order:

- [x] make all remaining unchanged lines conform to the new 79 character line length limit
- [x] reconsider the import structure and style to make it more uniform
- [x] move major explanatory parts of the documentation to rst files; limit documentation in py files to docstrings
- [x] ~~add CI tests for building FastF1 and testing the built package (catch things like missing packages in setup.cfg)~~
- [x] fix docstrings for various `Laps.pick_*` functions (confusing naming of vars in examples)

**Comment by theOehrly** [comment]
> What docstring improvements do you have in mind? I am already reworking some of those functions for #376 so I can take care of that as well.

The short example code snippet in many `Laps.pick_*` functions use a weird naming convention. For example `some_drivers_laps = ff1.pick_teams(['Mercedes', 'Williams'])` , but `ff1` is commonly used as abbreviation for the module, naming it `laps` would make more sense. This might be confusing, especially for newcomers.

**Comment by Casper-Guo** [comment]
I see. I will do some rewriting in my next commit.

**Comment by Casper-Guo** [comment]
With regards to input structure/style, have you investigated adding isort to pre-commit?

**Comment by theOehrly** [comment]
> With regards to input structure/style, have you investigated adding isort to pre-commit?

Yes, I probably want to add flake8-isort or flake8-import-order.

But I want "fix" all existing imports before that.

And I want to look into what a reasonable module structure is, to prevent circular imports (I problem that I occassionally have). This is not directly related but interesting for potential future additions and refactoring.
